### ISSUE TITLE
Double dollar "$$" is being converted to one "$"

### ISSUE THREAD

**Issue by guigs** [tulios/json-viewer#83]
"$$" is being coverted to "$"

raw: 

    [
     {"price_range":2,"label":"$$"},
     {"price_range":3,"label":"$$$"},
     {"price_range":4,"label":"$$$$"}
    ]

output:

```
[
  {
    "price_range": 2,
    "label": "$"
  },
  {
    "price_range": 3,
    "label": "$$"
  },
  {
    "price_range": 4,
    "label": "$$"
  }
]

```

**Comment by tulios** [comment]
Thanks @guigs.

Just for reference:
![screen shot 2016-05-11 at 23 42 51](https://cloud.githubusercontent.com/assets/33231/15197517/1cbad662-17d2-11e6-8a19-81b351f4f97b.png)

**Comment by tulios** [comment]
Fix released in version 0.13.1.

Thanks
